src/index.js

```diff
@@ -0,0 +1,145 @@
+  state.city = '';
+  cityName.textContent = state.city;
+  updateCity.value = state.city;
+};
+
+const tempColorAndLandscape = () => {
+  const tempColorElement = document.getElementById('tempValue');
+  const landscape = document.getElementById(
+    'weather-garden-containing-landscape'
+  );
+
+  if (state.temp <= 49) {
+    tempColorElement.style.color = 'teal';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (50 <= state.temp && state.temp <= 59) {
+    tempColorElement.style.color = 'green';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (60 <= state.temp && state.temp <= 69) {
+    tempColorElement.style.color = 'yellow';
+    landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (70 <= state.temp && state.temp <= 79) {
+    tempColorElement.style.color = 'orange';
+    landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else {
+    tempColorElement.style.color = 'red';
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  }
+};
+
+const changeTemp = (e) => {
+  const tempContainer = document.querySelector('.temperature');
+
+  if (e.target.id == 'tempUp') {
+    state.temp += 1;
+    console.log(state);
+    showTemperature();
+  } else if (e.target.id == 'tempDown') {
+    state.temp -= 1;
+    showTemperature();
+  }
+};
+
+const changeSky = () => {
+  let currentSky = document.getElementById('skys').value;
+  let skyContainer = document.getElementById('sky-garden');
+
+  state.sky = currentSky;
+  if (state.sky === 'cloudy') {
+    skyContainer.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (state.sky === 'sunny') {
+    skyContainer.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  } else if (state.sky === 'rainy') {
+    skyContainer.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (state.sky === 'snowy') {
+    skyContainer.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+};
+
+// const getWeather = async () => {
+//   let latitude, longitude;
+
+//   await axios
+//     .get('http://127.0.0.1:5000/location', {
+//       params: {
+//         q: state.city,
+//       },
+//     })
+//     .then((response) => {
+//       latitude = response.data[0].lat;
+//       longitude = response.data[0].lon;
+//     })
+//     .catch((error) => {
+//       console.log('error in finding location!');
+//     });
+
+//   axios
+//     .get('http://127.0.0.1:5000/weather', {
+//       params: {
+//         lat: latitude,
+//         lon: longitude,
+//       },
+//     })
+//     .then((response) => {
+//       // console.log(response.data);
+//       const kelvinTemperature = response.data.main.temp;
+//       // console.log(kelvinTemperature);
+//       const fahrenheitTemperature = Math.round(
+//         (kelvinTemperature - 273.15) * 1.8 + 32
+//       );
+//       state.temp = fahrenheitTemperature;
+//       tempValue.innerText = `${state.temp + '\u00B0F'}`;
+//       tempColorAndLandscape();
+//     })
+//     .catch((error) => {
+//       console.log('error in finding temperature!', error);
+//     });
+// };
```

Comment:
This is so close to working!  I think with a few changes (although I didn't test it) you could get it to work.  The big change is moving the second axios call to a separate function, and then calling that function inside the `.then` handler of the first axios call:
```suggestion
const getWeather = () => {
  //let latitude, longitude;

  axios
    .get('http://127.0.0.1:5000/location', {
      params: {
        q: state.city,
      },
    })
    .then((response) => {
      const latitude = response.data[0].lat;
      const longitude = response.data[0].lon;
      getTempForLocation(latitude, longitude);

    })
    .catch((error) => {
      console.log('error in finding location!');
    });
};
const getTempForLocation = (latitude, longitude) => {

  axios
    .get('http://127.0.0.1:5000/weather', {
      params: {
        lat: latitude,
        lon: longitude,
      },
    })
    .then((response) => {
      // console.log(response.data);
      const kelvinTemperature = response.data.main.temp;
      // console.log(kelvinTemperature);
      const fahrenheitTemperature = Math.round(
        (kelvinTemperature - 273.15) * 1.8 + 32
      );
      state.temp = fahrenheitTemperature;
      tempValue.innerText = `${state.temp + '\u00B0F'}`;
      tempColorAndLandscape();
    })
    .catch((error) => {
      console.log('error in finding temperature!', error);
    });
};
```